src/pages/templates/pageSections/hero/imageWithTextAndCTA.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Button, Image, Text, VStack, useBreakpointValue} from "@chakra-ui/react";
+
+export default function ImageWithTextAndCTA(){
+
+  const height = useBreakpointValue({ base: "250px", md: "400px" })
+  const btnSize = useBreakpointValue({ base: "md", md: "lg" })
+  const fontSize = useBreakpointValue({ base: "2xl", md: "4xl" })
+  const pl = useBreakpointValue({ base: "1rem", md: "3.5rem" })
```

Comment:
I would prefer some more readable names here, or just inline it. Because otherwise its hard to read where the height or font size is used (Could be Box, Image etc. height, oder the fontSize of the heading or the text)

Reply:
Alright I guess I'll just rename the variables then.

Reply:
I went with these names : heroHeight, ctaSize, heroTextSize, heroTextPadding

Reply:
Let me know what you think

Reply:
Sounds good to me!

Did you already commit these changes?

Reply:
Not yet, I'll do it right now

Reply:
The changes are commited